.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           cd ../edx-platform
           git remote add mitodl https://github.com/mitodl/edx-platform.git
           git fetch mitodl
-          git checkout mitodl/mitx/lilac
+          git checkout mitodl/arslan/fix-pipeline-import
```

Comment:
do we have maple branch ready to use here?

Reply:
We do have a Maple release but that would not put any impact on fixing the tests as I mentioned in the reviewer notes about why. https://github.com/mitodl/edx-sysadmin/pull/57#issuecomment-976476851